src/components/button/code/less/_button.less

```diff
@@ -63,7 +63,7 @@
 .button-variant(@background; @border) {
   background-color: @background;
       border-color: @border;
-             color: contrast(@background, #414141, #ffffff, 60%);
+             color: contrast(@background, #414141, #ffffff, 70%);
```

Comment:
I know it's out of the scope of this fix, but at some point can we use less variables for the magic values above? It would be more self-descriptive and fixes like this would only need to be done in one place.